my-code/movie-search/src/Containers/MovieDetails/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { fetchMovie } from '../../Actions';
+
+class MovieDetails extends Component {
+  componentDidMount() {
+    const { imdbID } = this.props.match.params;
+    this.props.fetchMovie(imdbID);
+
+    console.log(this.props);
```

Comment:
Careful with the console.logs and commented code (assuming that this submission is like a PR to production)

Reply:
Duly noted!